cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -164,6 +163,15 @@
     </description>
   </property>
 
+  <property>
+    <name>preview.enabled</name>
+    <value>true</value>
```

Comment:
Yes. We will add a configuration that UI will show the preview capability optionally.